cMake/FreeCAD_Helpers/SetGlobalCompilerAndLinkerSettings.cmake

```
@@ -90,4 +90,8 @@ macro(SetGlobalCompilerAndLinkerSettings)
             link_libraries(-lgdi32)
         endif()
     endif(MINGW)
+
+    # Enable the Topological Naming Problem mitigation code
+    add_compile_options(-DFC_ENABLE_TNP_FIX)
```

Shouldn't this be
```suggestion
    add_compile_options(-DFC_USE_TNP_FIX)
```
?  🤨 

---

Yes, yes it should! Thanks!